src/makeCalculator.js

```diff
@@ -37,7 +37,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(firstOperand, secondOperand) {
+      const operationResult = firstOperand + secondOperand;
+
+      return operationResult;
```

Comment:
we can just return firstOperand + secondOperand, without creating additional variable